pytorch_toolkit/ote/ote/modules/config_transformers/nncf_config_transformer.py

```diff
@@ -0,0 +1,115 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from copy import copy
+from mmcv import Config
+
+from ote.utils import load_config
+from ..registry import CONFIG_TRANSFORMERS
+from .base import BaseConfigTransformer
+from .utils import merge_dicts_and_lists_b_into_a
+
+def _convert_value_to_bool(v, key):
```

Comment:
don't reinvent the wheel, yaml is good at parsing (True, true, on, Off)

Reply:
Ok. Fixed